### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (footer)

### ISSUE THREAD

**Issue by A11Ybot-OP** [bazelbuild/bazel#14488]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Bazel - a fast, scalable, multi-language and extensible build system" - Bazel](https://bazel.build/)
#### Element path
.col-md-4.col-sm-12:nth-child(3) > blockquote > footer
#### Snippet
`<footer><cite>Michal Witkowski</cite>, Principal Tech Lead</footer>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 4.29 (foreground color: #777777, background color: #fafafa, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62

**Comment by fweikert** [comment]
We've migrated to a new site at https://bazel.build that fixes several accessibility issues.
